apps/mark/backend/src/app.ts

```diff
@@ -10,57 +14,74 @@ import {
   PrecinctId,
   safeParseElectionDefinition,
 } from '@votingworks/types';
-import { ScannerReportData, ScannerReportDataSchema } from '@votingworks/utils';
+import {
+  BooleanEnvironmentVariableName,
+  isFeatureFlagEnabled,
+  ScannerReportData,
+  ScannerReportDataSchema,
+} from '@votingworks/utils';
 
 import { getMachineConfig } from './machine_config';
 
+function constructAuthMachineState(input: {
+  electionHash?: string;
+}): InsertedSmartCardAuthMachineState {
```

Comment:
Perhaps this is another place where `input` is unnecessary, since currently there's not much else in this function to distinguish from. But with outstanding TODOs that might change, so I'll leave it up to you

Reply:
Agreed - went ahead and changed! https://github.com/votingworks/vxsuite/pull/3150/commits/cc7f767279b4cb457fe2f4365ee0ff4cca84f0a5